.github/workflows/build-test.yml

```diff
@@ -59,7 +65,7 @@ jobs:
          ./tools/ci/build-test-freebsd.sh
 
   build-test-windows:
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
Had to be done since windows was not compiling anymore due to an internal vcpkg library bug. See:  https://github.com/microsoft/vcpkg/issues/15035#issuecomment-742427969

Reply:
@dkroenke your suggestion worked!